pkg/motor/resource.go

```diff
@@ -58,6 +71,20 @@ func (r *motorResources) StoreWhatIs(whatIs *st.WhatIs) (*st.SchemaDefinition, e
 	return definition, nil
 }
 
+func (r *motorResources) StoreWhereIs(whereis *bt.WhereIs) bool {
+	_, ok := r.whatIsStore[whereis.Did]
```

Comment:
Did you mean to check `ok` before storing the whereis?

Reply:
my thinking was the cache should be updated regardless on call and the boolean is returned to indicate if the cache was already present

Reply:
Oh I see, resolving.